sources/packages/web/src/components/aest/OfferingApplicationBanner.vue

```diff
@@ -1,39 +1,51 @@
 <template>
-  <banner class="mb-2" :type="BannerTypes.Warning" :header="bannerText" />
+  <banner
+    class="mb-2"
+    :type="BannerTypes.Warning"
+    :header="bannerText"
+    :summary="bannerSummary"
+  />
 </template>
 <script lang="ts">
-import { onMounted, ref, computed, defineComponent } from "vue";
+import { ref, defineComponent, watchEffect } from "vue";
 import { PrecedingOfferingSummaryAPIOutDTO } from "@/services/http/dto";
-import { EducationProgramOfferingService } from "@/services/EducationProgramOfferingService";
 import { BannerTypes } from "@/types/contracts/Banner";
 export default defineComponent({
   props: {
     offeringId: {
       type: Number,
       required: true,
     },
+    precedingOfferingApplicationsCount: {
+      type: Number,
+      required: false,
+    },
   },
   setup(props) {
     const precedingOffering = ref({} as PrecedingOfferingSummaryAPIOutDTO);
-    const bannerText = computed(() => {
-      if (precedingOffering.value?.applicationsCount > 1) {
-        return `There are ${precedingOffering.value.applicationsCount} financial aid applications with this offering.`;
+    const bannerText = ref("");
+    const bannerSummary = ref("");
+    watchEffect(() => {
+      if (props.precedingOfferingApplicationsCount) {
+        if (props.precedingOfferingApplicationsCount > 1) {
+          bannerText.value = `There are ${precedingOffering.value.applicationsCount} financial aid applications with this offering.`;
+        }
+        bannerText.value =
+          props.precedingOfferingApplicationsCount === 1
+            ? "There is 1 financial aid application with this offering."
+            : "There are no financial aid applications with this offering.";
```

Comment:
line 30 condition will also fall into the 2nd statement for this conditional statement, right? let me know if I am wrong

Reply:
not sure, why this file needs a change for this PR

Reply:
Reverted back the changes.